### ISSUE TITLE
Pasting raw data from backup pdf says unsupported file format when trying to create a wallet? Also QR scanner doesnt work

### ISSUE THREAD

**Issue by pappyg45** [cryptoadvance/specter-desktop#913]
Whats the point of the backup pdf if the scanner and data format wont let you create a wallet?

**Comment by LaserHodl** [comment]
This seems pretty serious.

@pappyg45 can you provide the versions of Specter as well as the type of wallet and firmware version?

**Comment by pappyg45** [comment]
Specter ver 1.0.0. coldcard ver 3.1.9

**Comment by LaserHodl** [comment]
Can you make sure this item in the FAQ doesn't fix it? 

https://github.com/cryptoadvance/specter-desktop/blob/master/docs/faq.md

**Comment by LaserHodl** [comment]
Another potentially relevant FAQ - https://github.com/cryptoadvance/specter-desktop/blob/master/docs/faq.md#backup-files-not-showing-when-trying-to-load-backups

**Comment by LaserHodl** [comment]
If these don't work I'm sure the devs will have a look.

**Comment by pappyg45** [comment]
Yes none of those seem to fix the issue. The flow is Add new wallet > Import from wallet software > Enter wallet data > Continue

**Comment by LaserHodl** [comment]
1-on-1 wallet on a Coldcard?

**Comment by pappyg45** [comment]
> 1-on-1 wallet on a Coldcard?

Not sure what that means but I tried to import a new key from my SD card and the prints match and still unable to verify a TX

**Comment by ben-kaufman** [comment]
Hi @pappyg45 not sure I understand the issue exactly. Anyway if that's resolved I think you could just close the issue. Or if you think that's a problem with Specter would be helpful if you could please elaborate on the issue.

**Comment by pappyg45** [comment]
> Hi @pappyg45 not sure I understand the issue exactly. Anyway if that's resolved I think you could just close the issue. Or if you think that's a problem with Specter would be helpful if you could please elaborate on the issue.

Is there a way to automate this during import so the user doesn't need to remember to enter it manually for Coldcard? Is quite a scare when your funds are missing?

**Comment by ben-kaufman** [comment]
Automate what exactly? I didn't quite understand what were you trying to do and what was the issue?

**Comment by pappyg45** [comment]
> Automate what exactly? I didn't quite understand what were you trying to do and what was the issue?

So when I tried to copy / paste the backup pdf raw data it said that wallet already existed even though I created a wallet same seed different account number while trying to find my funds.  It didnt recognize the account number difference.

**Comment by ben-kaufman** [comment]
Which account wallet did that backup file belong to, the account with the funds or another account?
